src/services/player/autoPlayer.js

```diff
@@ -0,0 +1,36 @@
+// export default
+class AutoPlayer {
+  constructor() {
+    this.id = 'AutoPlayer';
+  }
+
+  generateAutoPlayerAnswer(min, max) {
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.floor(Math.random() * (max - min + 1) + min);
+  }
+
+  getAnswer(image, answers) {
+    // console.log(image);
+    // console.log(answers);
+    if (typeof image !== 'string') {
+      throw new Error('image type error');
+    }
+    if (!Array.isArray(answers)) {
```

Comment:
are you sure about this check?

plase try something like this in developer console/js playground

```
const myArr = ['a', 2]
const myString = 'test'

console.log(Array.isArray(myArr))
console.log(Array.isArray(myString))

console.log(!Array.isArray(myArr))
console.log(!Array.isArray(myString))

console.log(!!Array.isArray(myArr))
console.log(!!Array.isArray(myString))
```

and look for the results while you change flag `!` `!!`

Reply:
![Screenshot 2021-12-16 183249](https://user-images.githubusercontent.com/21975991/146420705-4773f94e-3d7c-4f44-96c4-162b9381cc78.png)
Works as expected.